build_tools/python/e2e_test_artifacts/common_artifacts.py

```diff
@@ -0,0 +1,54 @@
+## Copyright 2022 The IREE Authors
```

Comment:
Are there any other types of artifacts going to use this module? It looks like the usage now is most about `model_artifacts` instead of `common_artifacts`. Maybe it should be named as such?

Reply:
Good idea. Done